templates/contribute/style.md

```diff
@@ -282,184 +278,221 @@ example (n : ℝ) (h : 1 < n) : 0 < n := by linarith
 is preferred over
 
 ```lean
-example (n : ℝ) : 1 < n → 0 < n := λ h, by linarith
+example (n : ℝ) : 1 < n → 0 < n := fun h ↦ by linarith
 ```
 
 and
 
 ```lean
-example (n : ℕ) : 0 ≤ n := dec_trivial
+example (n : ℕ) : 0 ≤ n := Nat.zero_le n
```

Comment:
Isn't there a Lean 4 analogue of `dec_trivial`? I know `decide` is slightly different.

Reply:
I think `decide` *is* supposed to be the Lean 4 analogue of `dec_trivial`, but it doesn't seem to handle free or meta variables (at least, that's the error message I get when I try `by decide`).